tests/h/presenters/group_json_test.py

```diff
@@ -81,6 +84,46 @@ def test_open_group_asdict_with_links_svc(self, factories, links_svc):
 
         links_svc.get_all.assert_called_once_with(group)
 
+    def test_it_does_not_expand_by_default(self, factories):
+        group = factories.OpenGroup(name='My Group',
+                                    pubid='mygroup')
+        presenter = GroupJSONPresenter(group)
+
+        model = presenter.asdict()
+
+        assert model['organization'] == ''
```

Comment:
As noted above, I would expect the `organization` field to be `null` if there is no organization.